### ISSUE TITLE
Random strings appearing in code blocks

### ISSUE THREAD

**Issue by erroneousio** [RocketChat/Rocket.Chat#1860]
Rocket.Chat version: 0.12.1
Running on Ubuntu 14.04 x64

Wanted to show a friend a command that I stumbled on and because formatting is cool, I put it in between back-ticks for the markdown formatting. I entered:

```
`grep --color -E 'pattern|$'`
```

Pushing enter, I received something I didn't expect:
```
grep --color -E 'pattern|$NqQJTs2zDQqdnw6kf$#39;
```

Some tinkering in the conversation narrowed the issue down to the dollar sign and inverted comma in a code block.

```
`$'`
```

Gives:

```
$srJLrbcum38dGMc4n$#39;
```

This is random and changes each submission and when you edit the message. I'm guessing it's an ID or something along those lines.

**Comment by erroneousio** [comment]
Updated to latest version of Rocket.Chat server and bug is still present.

**Comment by geekgonecrazy** [comment]
Looks like will be closed in #1869 :)
